src/storage/hummock_sdk/src/compaction_group/mod.rs

```diff
@@ -272,24 +274,24 @@ pub mod group_split {
     }
 
     /// Merge the right levels into the left levels.
-    pub fn merge_levels(left_levels: &mut Levels, right_levels: Levels) {
+    pub fn merge_levels(
+        left_levels: &mut Levels,
+        right_levels: Levels,
+        max_sub_level_id: Option<u64>,
+    ) {
         let right_l0 = right_levels.l0;
 
-        let mut max_left_sub_level_id = left_levels
-            .l0
-            .sub_levels
-            .iter()
-            .map(|sub_level| sub_level.sub_level_id + 1)
-            .max()
-            .unwrap_or(0); // If there are no sub levels, the max sub level id is 0.
-        let need_rewrite_right_sub_level_id = max_left_sub_level_id != 0;
+        let mut next_right_sub_level_id = max_sub_level_id
```

Comment:
The merge operation is still in a ‘ready’ state, can we remove this option and compute it externally?

Or when will merge_levels, receive a None ?

Reply:
Fixed. In-mem max_sub_level_id's type is modified to u64.